libs/arcade-cli/arcade_cli/main.py

```diff
@@ -271,16 +271,11 @@ def mcp(
             handle_cli_error("Failed to run MCP server")
 
     except KeyboardInterrupt:
-        console.print("\n[yellow]MCP server stopped[/yellow]")
-        raise typer.Exit(0)
+        console.print("\n[yellow]MCP server gracefully shutdown[/yellow]")
     except FileNotFoundError:
-        console.print(
-            "[red]arcade_mcp_server module not found. Make sure arcade-mcp-server is installed.[/red]"
+        handle_cli_error(
```

Comment:
As part of this PR I improved how our CLI commands are handling errors to ensure the correct data is sent to posthog